### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js [hunk 1]
```diff
@@ -27,8 +26,8 @@ var appContext = $("#details").data("appcontext");
 var marker_1 = appContext + '/public/iot.unit.device.stats/images/map-marker-1.png';
```
Comment (harshanL) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Use correct camel-notation.

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js [hunk 2]
```diff
@@ -88,41 +83,97 @@ $(document).ready(function () {
 
 function updateGraphs() {
     var tv = 5000;
-
-    var fields = [];
+    var graphs = {};
     for (var op in monitor_operations) {
-        if (monitor_operations[op].name == 'gps') {
+        var op_name = monitor_operations[op].name;
```
Comment (harshanL) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Use correct camel-notation.

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js [hunk 3]
```diff
@@ -88,41 +83,97 @@ $(document).ready(function () {
...
+        if (op_name == 'gps') {
             $('#map').removeClass('hidden');
         } else {
-            fields.push({name: monitor_operations[op].name});
+            var x_label = '', y_label = '';
```
Comment (harshanL) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Use correct camel-notation.

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js (hunk 1 above)
Comment (charithag) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Fix with PR https://github.com/wso2/carbon-device-mgt-plugins/pull/206

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js (hunk 2 above)
Comment (charithag) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Fix with PR https://github.com/wso2/carbon-device-mgt-plugins/pull/206

File: components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js (hunk 3 above)
Comment (charithag) [inline components/device-mgt-iot/org.wso2.carbon.device.mgt.iot.ui/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/public/js/device-stats.js]: Fix with PR https://github.com/wso2/carbon-device-mgt-plugins/pull/206

**Pull request merged**